cli/azd/cmd/auth_login.go

```diff
@@ -466,6 +466,8 @@ func (la *loginAction) login(ctx context.Context) error {
 		if err != nil {
 			return fmt.Errorf("logging in: %w", err)
 		}
+	} else if auth.ShouldUseCloudShellAuth() {
+		return fmt.Errorf("logging in: already logged in using CloudShell")
```

Comment:
Just wondering - should we consider this to be an error, or should we just have `azd login` return success in this case?  It feels like the `login` operation didn't fail at some level here. `azd` is authenticated in this case, right?

Reply:
Indeed it would not be an error. This will also change as we're following the behavior of `az` CLI and offering the user the option to log in with a device token when they run `azd auth login` in Cloud Shell.